Sources/ArcGISToolkit/Components/OfflineMapAreasView/Utilities/LoadableImageView.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+/// A view that loads a `LoadableImage` and displays it.
+/// While the image is loading a progress view is displayed.
+/// If there is an error displaying the image a red exclamation circle is displayed.
+struct LoadableImageView: View {
```

Comment:
If this is the same view as the one from AuthenticationExample app, maybe log an issue so we can remove the duplicated code from that example app later.

Reply:
Yes this is a duplicate.

Reply:
Hmm. Maybe I left a wrong suggestion here. The new `package` access modifier won't help in this case, as the example app is a standalone project 🤔  Lemme think about it again…

Reply:
Actually there is an issue for this `swift/issues/3218`

Reply:
maybe we should implement `3218` as part of the offline toolkit component.